### DISCUSSION TITLE
The purpose of this discussion is to ask some questions about 0-D Atmospheric Modeling (F0AM) v3.1.

### DISCUSSION THREAD

**Discussion by GeAntong1** [AirChem/F0AM#21] [Q&A]
Question 1

Is there a direct relationship between ozone generation rate and solar irradiance in F0AM? Is there a formula that can describe the relationship between the ozone generation rate and solar irradiance.

Question 2

What is the value of solar radiation corresponding to photon flux (photons/cm2/s/nm) in the ExampleLightFlux.txt file in the simulation ozone generation example? Only ExampleLightFlux.txt was found in the file package, but no documents related to solar irradiance were found.

Question 3

Is there a positive correlation between solar irradiance and photon flux? Can it be expressed in a formula?

**Comment by gmwolfe** [comment]
Hi Ge,

Please post this in the F0AM google group, as instructed in the ReadMe. We'd like to keep all the discussions in one place.

Thank you,

~Glenn
